coderd/prometheusmetrics/aggregator_test.go

```diff
@@ -92,7 +114,7 @@ func verifyCollectedMetrics(t *testing.T, expected []agentsdk.AgentMetric, actua
 		err := actual[i].Write(&d)
 		require.NoError(t, err)
 
-		require.Equal(t, "agent_name", *d.Label[0].Name)
+		require.Equal(t, "agent_name", *d.Label[0].Name, d.String())
 		require.Equal(t, testAgentName, *d.Label[0].Value)
 		require.Equal(t, "username", *d.Label[1].Name)
 		require.Equal(t, testUsername, *d.Label[1].Value)
```

Comment:
Adding `d.String()` to the `require()` call here is probably not a bad idea here too.

Reply:
Fixed